frontend/packages/ux-editor/src/components/TaskNavigation/TaskCard.module.css

```
@@ -9,7 +9,8 @@
   gap: var(--fds-spacing-3);
 }
 
-.details > * {
+.details > *,
+.details h1 {
```

I'm wondering if this has any drawbacks? If we could use descendant with wildcard without problems here, we would not have to specify h1 manually, in case it would for example change down the line?

```suggestion
.details * {
```

---

nice that is much better! 👍